packages/nimble-components/src/rich-text/mention-listbox/index.ts

```diff
@@ -108,9 +108,11 @@ export class RichTextMentionListbox extends FoundationListbox {
      * @public
      */
     public show(options: MentionListboxShowOptions): void {
-        const listboxTop = options.anchorNode.getBoundingClientRect().bottom;
+        const anchorRect = options.anchorNode.getBoundingClientRect();
+        const availableSpaceBelow = window.innerHeight - anchorRect.bottom;
+        const availableSpaceAbove = anchorRect.top;
         this.availableViewportHeight = Math.trunc(
-            window.innerHeight - listboxTop
+            Math.max(availableSpaceAbove, availableSpaceBelow)
```

Comment:
Talked offline but biggest concern is making sure discussion is captured about how these changes here align with out other listbox / anchored region implementations. Want alignment to increase generally or have clear reasons why they are different

Reply:
Updated the PR description. Essentially, we're making the mention listbox more consistent with the other dropdown controls with this change.

Reply:
That description is not conducive to quick review 😅. Could you break it into bullets for the controls referenced and then link to relevant source. Goal is to be able to quickly see that this is aligning or misaligned with how our other controls behave.

Reply:
What, you don't trust me? 😅 Fine, fine.... Reorganized description and added all the relevant source links.